molecule/default/requirements.yml

```diff
@@ -1,2 +1,7 @@
 ---
-- src: ome.nfs_mount
+- name: nfs_mount
+  src: https://github.com/pwalczysko/ansible-role-nfs-mount/
+  version: Rocky9
+- name: nfs_share
+  src: https://github.com/pwalczysko/ansible-role-nfs-share/
```

Comment:
why?

Reply:
could you point to ome

Reply:
> could you point to ome

Done in https://github.com/ome/ansible-role-nfs-share/pull/7/commits/b59fb58e1f59325c103d2dc0892af5cc01fa3a3e, ta